### PR TITLE
ETL-3185.  Copied a method to read key value pair type conf files

### PR DESCRIPTION
**Pull request by hema1024**
Copied from Q2

### CODE REVIEW

**Comment by ffan07039** [comment]
:+1:

File: src/main/java/com/integralads/etl/utils/mapreduce/MRUtil.java [hunk 1]
```diff
@@ -87,5 +94,31 @@ private static String removeSchemeAndAnchors(URI uri) {
         return paths;
     }
 
+    public static Map<String, String> parseKeyValueConfFile(Path configFile) throws Exception {
+
+        BufferedReader fis = new BufferedReader(new FileReader(configFile.toString()));
```
Comment (ycheng-ias) [inline src/main/java/com/integralads/etl/utils/mapreduce/MRUtil.java]: Any reason that we cannot use System.Properties.load() method?

**Comment by ycheng-ias** [comment]
:+1:

**Comment by hema1024** [comment]
Actually, I think using Properties.load() is a cleaner way.  Since it is a one liner I've removed the new method.

**Pull request merged**